terraform/deployments/govuk-test/main.tf

```diff
@@ -37,6 +37,8 @@ module "govuk" {
   mongodb_host                  = "mongo-1.test.govuk-internal.digital,mongo-2.test.govuk-internal.digital,mongo-3.test.govuk-internal.digital"
   redis_host                    = "pink-backend-redis.0f3erf.ng.0001.euw1.cache.amazonaws.com"
   statsd_host                   = "statsd.test.govuk-internal.digital"
+  signon_db_url                 = "mysql2://root:root@mysql-primary.test.govuk-internal.digital/signon"
```

Comment:
I wonder if this should be a secret as it encodes a username and password?

Reply:
+1 Also ideally we'd make just the password a secret so that it's still clear what's connecting where. (Is it actually set to `root:root`? 🙈)